clang/test/CodeGenSYCL/kernel-op-calls.cpp

```diff
@@ -0,0 +1,94 @@
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -triple spir64-unknown-unknown -disable-llvm-passes -sycl-std=2020 -emit-llvm -o - %s | FileCheck %s
```

Comment:
I am in the process of adding more test cases, fixing the CHECK line, adding title etc, but I'd like to get some early feedback on the approach.